src/plugins/tscn/tscnplugin.cpp

```diff
@@ -574,6 +648,10 @@ static void writeTileset(const Map *map, QFileDevice *device, bool isExternal, A
             sanitizeQuotedString(it.key()),
             sanitizeQuotedString(it->id)));
     }
+
+    if (!isExternal)
+        writeExtObjects(device, assetInfo);
```

Comment:
I'll move it out of the function call. I think I just put it there because I was trying to match Godot's files, which put the Texture2D resources first, but the order doesn't actually matter.